src/View/Rules/index.html.twig

```diff
@@ -6,11 +6,15 @@
 {% block stylesheet %}
 	{{parent()}}
 	<link rel="stylesheet" href="/assets/css/rules.css">
+	<link rel="stylesheet" type="text/css" href="/assets/css/userForm.css">
+	<link rel="stylesheet" type="text/css" href="/assets/css/login.css">
 {% endblock %}
 
 {% block content %}
-	<main>
-		<h1 class="rule-title">Bienvenue dans Happiness & Glitters</h1>
+	<section>
```

Comment:
Please use a main tag instead of section or you might break the css // And it is mandatory for good referencing